plugin.yml

```diff
@@ -1,8 +1,10 @@
 name: Elevator
 main: ColinHDev\Elevator\Elevator
-version: "1.0"
+version: 1.0.1
 
-api: 3.0.0
+api: 4.0.0
+
+load: POSTWORLD
```

Comment:
Is that necessary?

Reply:
In future pocketmine version (like PM5 or maybe PM4(.1)) the `load` property will be required. I've only added this to check if MyPlot is on the server after MyPlot was loaded. (Plugins with loading POSTWORLD will be loaded at the last)